lib/src/error_group.dart

```diff
@@ -188,7 +188,7 @@ class _ErrorGroupFuture<T> implements Future<T> {
     _completer.future.catchError((_) {});
   }
 
-  Future/*<S>*/ then/*<S>*/(/*=S*/ onValue(/*=T*/ value), {Function onError}) {
+  Future/*<S>*/ then/*<S>*/(onValue(/*=T*/ value), {Function onError}) {
```

Comment:
The original declaration was intended. The way it's written now will not be runtime-strong-mode clean.